modules/webxr/register_types.cpp

```diff
@@ -45,8 +45,10 @@ void initialize_webxr_module(ModuleInitializationLevel p_level) {
 	GDREGISTER_ABSTRACT_CLASS(WebXRInterface);
 
 #ifdef WEB_ENABLED
-	webxr.instantiate();
-	XRServer::get_singleton()->add_interface(webxr);
+	if (XRServer::get_singleton()) {
+		webxr.instantiate();
+		XRServer::get_singleton()->add_interface(webxr);
+	}
```

Comment:
Just an optional nitpick, but I'd personally prefer this with a local variable like:

```suggestion
	XRServer *xr_server = XRServer::get_singleton();
	if (xr_server) {
		webxr.instantiate();
		xr_server->add_interface(webxr);
	}
```